src/RestApi/Controllers/ProductCategories.php

```diff
@@ -117,6 +117,27 @@ public function prepare_item_for_response( $item, $request ) {
 			'permalink'   => get_term_link( $item->term_id, 'product_cat' ),
 		);
 
+		if ( $request->get_param( 'show_review_count' ) ) {
+			global $wpdb;
+			$term_id       = $item->term_id;
+			$term_taxonomy = 'product_cat';
+
+			$products_of_category_sql = "
```

Comment:
We should use `wpdb->prepare` here to pass in `$item->term_id` as a decimal (`%d`). Since there is only one term, you can use `=` rather than `IN` for this which makes using prepare slightly easier.